src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -619,12 +631,6 @@ XElement CreateApplicationElement (XElement manifest, string applicationClass, L
 				throw new InvalidOperationException ("Application cannot have both a type with an [Application] attribute and an [assembly:Application] attribute.");
 
 			ApplicationAttribute appAttr = assemblyAttr.SingleOrDefault () ?? typeAttr.SingleOrDefault ();
-			var ull1 = usesLibraryAttr ?? Array.Empty<UsesLibraryAttribute> ();
-			var ull2 = typeUsesLibraryAttr.AsEnumerable () ?? Array.Empty<UsesLibraryAttribute> ();
-			var usesLibraryAttrs = ull1.Concat (ull2);
-			var ucl1 = usesConfigurationAttr ?? Array.Empty<UsesConfigurationAttribute>();
-			var ucl2 = typeUsesConfigurationAttr.AsEnumerable () ?? Array.Empty<UsesConfigurationAttribute> ();
-			var usesConfigurationattrs = ucl1.Concat (ucl2);
```

Comment:
It seems like this made lots of locals before, and we didn't really need them?

Reply:
Yup looks like this code above was just to concatinate the lists. But you already do that in the new foreach loop replacement.